karyon3-admin-simple/src/main/java/com/netflix/karyon/admin/SimpleHttpServer.java

```diff
@@ -0,0 +1,63 @@
+package com.netflix.karyon.admin;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+import javax.annotation.PreDestroy;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
```

Comment:
Yes, but that's fine.  The implementation of the HttpServer here is decoupled from the implementation of the admin endpoints.  We can easily port this to RxNetty or Jetty if so desired.  Since we are using Oracle/Sun JDK this helps minimize additional dependencies.